rest-api-client/src/main/java/com/stnetix/ariaddna/client/api/AriaddnaApi.java

```diff
@@ -43,16 +43,16 @@
 import java.util.List;
 import java.util.Map;
 
-public class DefaultApi {
```

Comment:
My concern here is https://groups.google.com/forum/#!topic/swagger-swaggersocket/WlR_0U7eLA0
May be we do not know some functionality in Swagger?